mc/Mongo-Core.package/MongoIsMaster.class/instance/initializeWith..st

```diff
@@ -0,0 +1,5 @@
+initialization
+initializeWith: aCollection 
```

Comment:
Is it a collection or dictionary?

Reply:
More particularly a dictionary, but when somebody presses on "create" in the debugger (in a DNU), the parameter has this name. This was defined in some method of the hierarchy, not sure if it's on purpose or a bug.